src/sandstorm/sealed.capnp

```diff
@@ -0,0 +1,29 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0xfe5c7cde99284f21;
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+using SystemPersistent = import "supervisor.capnp".SystemPersistent;
+
+interface SealedUiView @0x8ccbcb3c38aa1574 {}
```

Comment:
Ahhh, yes.  I'll add a comment explaining how this is only unsealable into a UiView by a user, not by a grain or other entity.

The goal was that we be able to share a reference to grain A without granting the intermediary grain B that holds the reference (currently a SealedUiView capability) the ability to do literally anything that UiView allows, like open new sessions for grain A impersonating other users.

Perhaps overloading the term `sealed` here was a poor choice.  If you have a better name for this, I'm definitely open to changing it.